bigframes/session/loader.py

```diff
@@ -525,21 +538,12 @@ def _read_bigquery_load_job(
                 )
         else:
             load_job = self._bqclient.load_table_from_file(
-                filepath_or_buffer, table, job_config=job_config
+                filepath_or_buffer, destination=table, job_config=job_config
             )
 
         self._start_generic_job(load_job)
         table_id = f"{table.project}.{table.dataset_id}.{table.table_id}"
 
-        # Update the table expiration so we aren't limited to the default 24
```

Comment:
~I assume `_storage_manager.create_temp_table` handles this in the anonymous dataset case now?~

Edit: Yes, I see we do set an expiration above:

```
        expiration = (
            datetime.datetime.now(datetime.timezone.utc) + constants.DEFAULT_EXPIRATION
        )
        table = bf_io_bigquery.create_temp_table(
            self.bqclient,
            self.allocate_temp_table(),
            expiration,
            schema=schema,
            cluster_columns=list(cluster_cols),
            kms_key=self._kms_key,
        )
```